src/js/utils/placement.ts

```diff
@@ -0,0 +1,4 @@
+export enum Placement {
```

Comment:
Is utils the best place for this? Do we have a config/objects type of folder location or something similar?

Reply:
Funny you should mention that! @bradjteamsnap and I were just talking about how in the V4 branch, we have a `types` folder.  Im okay with keeping it in utils on this branch since there isn't already a home, but in V4 it will live in `types` and, if we keep v3 around much longer, I am happy to move it to a more suitable home once we have one :)

Reply:
Sounds good